kmeans-sklearn/README.md

```diff
@@ -0,0 +1,57 @@
+# KMeans via sklearn (Python2)
+
+Two clustering examples using [sklearn](https://scikit-learn.org/stable/).
+
+If this is your first time using Binaris, visit our [Getting Started](https://dev.binaris.com/tutorials/python/getting-started/) page to set up your account for free.
+
+There are two functions in this example
+
+#### public_kmeans_image_quant_py2
+
+Uses kmeans to quantize an input image potentially reducing the number of unique colors greatly. This can allow for greater compression of the input image or pleasing filter-esque effects.
+
+#### public_kmeans_simple_py2
```

Comment:
what's the purpose of this example?
iiuc it does nothing. generates the model  during initialization and prints the same labels over and over when invoked. why not use it to identify an iris?

Reply:
This is meant to be an extremely simple example which just proves/shows a running kmeans prediction on the system. I chose to do it at initialization time so latency isn't effected outside of the first run.

Identifying an Iris means you know how to properly structure a query with Iris data. Would you rather I

1. Add a simple script that invokes the function with a valid 'iris' in the body
2. Remove the example altogether.